extensions/debug-server-ready/package.json

```diff
@@ -158,6 +158,50 @@
                         "default": false
                       }
                     }
+                  },
+                  {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "markdownDescription": "%debug.server.ready.serverReadyAction.description%",
+                    "default": {
+                      "action": "startDebugging",
+                      "config": {
+                        "type": "node",
+                        "request": "launch"
+                      },
+                      "killOnServerStop": false
+                    },
+                    "required": [
+                      "config"
+                    ],
+                    "properties": {
+                      "action": {
+                        "type": "string",
+                        "enum": [
+                          "startDebugging"
```

Comment:
Does this mean the only valid option for now is startDebugging?

Reply:
This is an extension of `serverReadyAction` to support a different type when using `startDebugging`. There are other options (the json schema is hard to parse here but they are there). `startDebugging` is the only one that is relevant in this scenario.